packages/uniforms-bridge-json-schema/__tests__/JSONSchemaBridge.ts

```diff
@@ -388,6 +388,22 @@ describe('JSONSchemaBridge', () => {
         expect(bridge.getError(name, error)).toEqual(error.details[0]);
       });
     });
+
+    it('works with correct error (complex instance paths - dots in names)', () => {
+      const pairs = [
+        ['["path.with.a.dot"]', '/path.with.a.dot'],
+        ['["path.with.a.dot"].another', '/path.with.a.dot/another'],
+        [
+          '["path.with.a.dot"].["another.with.a.dot"]',
+          '/path.with.a.dot/another.with.a.dot',
+        ],
+      ];
```

Comment:
Let's add a couple more tests:
```
a.["b"]
a.["b"].c
a.["b.c"]
a.["b.c"].d
["a"].b
["a.b"].c
```

Also, how does it work with `/` in the name?

Reply:
> Also, how does it work with `/` in the name?

I believe it's covered  by the other tests (like [this one](https://github.com/vazco/uniforms/blob/76181387fc065ceb153bc6dfc4c9cb61eb27e836/packages/uniforms-bridge-json-schema/__tests__/JSONSchemaBridge.ts#L366))